bindings/python/benchmarks/benchmarks.py

```diff
@@ -107,6 +117,13 @@ class ProfileRead:
     of reading MongoDB data.
     """
 
+    @staticmethod
+    def exercise_table(table):
```

Comment:
Please add a comment to explain why this is needed.

Reply:
Done.